source/whats-new.txt

```diff
@@ -17,6 +17,20 @@ What's New
 .. meta::
    :keywords: change, update, upgrade, compatible, backwards
 
+Changes in Version 1.6.0
+------------------------
+
+.. warning:: Breaking Change
+
+   This version of {+driver-short+} introduces the following breaking change:
+
+   - Removes support for Python v3.8
+
+- Adds support for PyArrow 18.0 and Python 3.13
+- Fixes support for ``null`` and extension types in nested data
```

Comment:
S: can you explain what this is fixing? It might be confusing to know if this is added support to fixing a serialization issue

Reply:
Got some clarification and updated this to be more clear